helix-core/src/test/java/org/apache/helix/integration/task/TestStuckTaskQuota.java

```diff
@@ -63,6 +63,9 @@ public void beforeClass() throws Exception {
 
   @AfterClass
   public void afterClass() throws Exception {
+    String testClassName = this.getShortClassName();
```

Comment:
As we can see, there are many changes like this. So let's do it in the parent class. Shall we?

Reply:
removed.